invoicing/enums.py

```diff
@@ -48,8 +48,10 @@ class InstallmentPercentageSpecifier(Enum):
     SALES_PRICE = "SALES_PRICE"
     DEBT_FREE_SALES_PRICE = "DEBT_FREE_SALES_PRICE"
     SALES_PRICE_FLEXIBLE = "SALES_PRICE_FLEXIBLE"
+    RIGHT_OF_OCCUPANCY_SALES_PRICE = "RIGHT_OF_OCCUPANCY_SALES_PRICE"
```

Comment:
I don't think term "sales price" has been used in HASO side, the price is "right of occupancy payment" in there, so `RIGHT_OF_OCCUPANCY_PAYMENT` would be a more precise name for this IMO.